pkgs/development/libraries/bullet/default.nix

```diff
@@ -51,10 +39,6 @@ stdenv.mkDerivation rec {
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       "-DOPENGL_FOUND=true"
```

Comment:
The `FIND_PACKAGE` we were previously removing will set this; it should be dropped. Also I wouldn’t be surprised if we can remove this whole section and just let it build everything now.